src/Raven.Server/Web/System/DatabasesHandler.cs

```diff
@@ -248,6 +238,23 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
+        private DynamicJsonValue GetNodeJson(string tag, ServerNode.Role role, LicenseLimits license, ClusterTopology clusterTopology, string database)
+        {
+            var json = new DynamicJsonValue
+            {
+                [nameof(ServerNode.Url)] = GetUrl(tag, clusterTopology),
+                [nameof(ServerNode.ClusterTag)] = tag,
+                [nameof(ServerNode.Database)] = database,
+                [nameof(ServerNode.ServerRole)] = role,
+            };
+
+            if (license != null && license.NodeLicenseDetails.TryGetValue(tag, out var nodeDetails))
+            {
+                json[nameof(ServerNode.LastServerVersion)] = nodeDetails.BuildInfo.GetCleanedFullVersion();
```

Comment:
I do not like that, introduce another property if needed called AssemblyVersion and push there RavenVersionAttribute.Instance.AssemblyVersion

Reply:
Also how often this is refreshed? when client will catch a change in this value?

Reply:
It gets the nodes version in the client first in 'FirstTopologyUpdate' method before it performs any requests..
and after that it updates the version of the node it send the request to (by the header 'AssemblyVersion' it gets in the response).. it updates the node version after every successfull request..